bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/types/StringListType.java

```diff
@@ -27,7 +27,7 @@
  * This type can be used for items that are dealing with telephony functionality.
  *
  * @author Thomas.Eichstaedt-Engelen / Initially CallType
- * @author Gaël L'hopital - port to Eclipse SmartHome
+ * @author Gaël L'hopital - port to openHAB
```

Comment:
Hm, I assume this cannot be changed as it will change the "history".
Gaël L'hopital has been ported it to Eclipse SmartHome, regardless if we migrate that code from Eclipse SmartHome to openHAB Core.